ecs_logging/_stdlib.py

```diff
@@ -1,5 +1,57 @@
 import logging
+import time
+import json
+from ._meta import ECS_VERSION
+from ._utils import deepmerge, de_dot
 
 
 class StdlibFormatter(logging.Formatter):
-    """TODO"""
+    """ECS Formatter for the standard library ``logging`` module"""
+
+    WANTED_ATTRS = {
+        "levelname": "log.level",
+        "funcName": "log.origin.function",
+        "lineno": "log.origin.file.line",
+        "filename": "log.origin.file.name",
+        "message": "log.original",
+        "name": "log.logger",
+    }
+
+    def __init__(self):
+        super(StdlibFormatter, self).__init__()
+        self.converter = time.gmtime
+
+    def format(self, record):
+        # type: (logging.LogRecord) -> str
+        result = self.format_to_ecs(record)
+        return json.dumps(result, sort_keys=True, separators=(",", ":"))
+
+    def format_to_ecs(self, record):
+        # type: (logging.LogRecord) -> dict
+        """Function that can be overridden to add additional fields
+        to the JSON before being dumped into a string.
+        """
+        timestamp = "%s.%03dZ" % (
+            self.formatTime(record, datefmt="%Y-%m-%dT%H:%M:%S"),
+            record.msecs,
+        )
+        result = {"@timestamp": timestamp, "ecs": {"version": ECS_VERSION}}
```

Comment:
I like the addition of `ECS_VERSION`. I had considered adding it to mine in Curator but forgot.